templates/include/telemetry/metricsData.1.schema.json

```diff
@@ -0,0 +1,21 @@
+"scalars": @TELEMETRY_SCALARS_1_JSON@,
+"keyedScalars": {
+  "type": "object",
+  "additionalProperties": {
+    "keyedScalar": @TELEMETRY_SCALARS_1_JSON@
```

Comment:
This section should not have a name, and should just look like `"additionalProperties":  @TELEMETRY_SCALARS_1_JSON@`